nixos/tests/docker-tools.nix

```
@@ -46,6 +46,18 @@ let
         echo 'runAsRoot has run.'
       '';
     };
+
+  chownTestImage =
+    pkgs.dockerTools.streamLayeredImage {
+      name = "chown-test";
+      tag = "latest";
+      enableFakechroot = true;
+      fakeRootCommands = ''
+        touch testfile
+        chown 12345:12345 testfile
+      '';
+      config.Entrypoint = [ "${pkgs.coreutils}/bin/stat" "-c" "%u:%g" "/testfile" ];
```

```suggestion
      config.Cmd = [ "${pkgs.coreutils}/bin/stat" "-c" "%u:%g" "/testfile" ];
```

I believe this is more idiomatic.